spark/src/main/scala/org/apache/druid/spark/mixins/Logging.scala

```diff
@@ -0,0 +1,123 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.spark.mixins
+
+import org.slf4j.{Logger, LoggerFactory}
+
+/**
+  * Simplified version of org.apache.spark.internal.Logging.
+  */
+trait Logging {

```

Comment:
Is there a reason why we can't use `org.apache.druid.java.util.common.logger.Logger` class?

Reply:
Done. We lose the warn and error guards but that seems worth it to be able to rely on the Druid abstraction.